Dockerfile

```diff
@@ -18,8 +18,6 @@ RUN chmod +x /entrypoint.sh
 RUN chmod +x /sast_to_log.py
 RUN chmod +x /main.py
 
```

Comment:
```suggestion

# We are not setting USER because this image is only run on github.com.
#   A similar approach is used in https://github.com/super-linter/super-linter/blob/main/Dockerfile
```

When you don't adopt best practices (e.g., setting `USER`) you should provide an explanation. 

Otherwise, check whether if it is possible to run the action with the specific github-action user.